lib/test.py

```diff
@@ -5,6 +5,7 @@
 import re
 import shutil
 import sys
+import time
```

Comment:
Re commit message of 'Add tests durations collecting':

> Using 'WorkerTaskResult' message queue

Again, it is not a queue.

Reply:
Right, corrected.